client/scala/src/main/scala/ai/verta/repository/Commit.scala

```diff
@@ -72,21 +76,49 @@ class Commit(
     if (saved)
       Failure(new IllegalCommitSavedStateException("Commit is already saved"))
     else
-      blobsList().flatMap(list => createCommit(message = message, blobs = Some(list)))
+      loadBlobs().flatMap(_ => {
+        val toVersion = blobs
```

Comment:
```suggestion
        val blobsToVersion = blobs
```
I recommend giving things meaningful names.

Reply:
Fixed.